src/gui/qgsmetadatawidget.h

```diff
@@ -43,16 +43,50 @@ class GUI_EXPORT QgsMetadataWidget : public QWidget, private Ui::QgsMetadataWidg
 
     /**
      * Constructor for the wizard.
+     * \note
+     * For use with Layer. Sets mMetadata if pointer is valid
+     *  calls setMetadata, using mMetadata
+     * \param layer to set the main QgsLayerMetadata with mLayer->metadata() when not nullptr
+     * \see mMetadata
+     * \see setMetadata
+     * \since QGIS 3.0
```

Comment:
Our convention is not to include "\since" tags if the version matches the version the class itself was added - so the "\since QGIS 3.0" lines can be dropped.

Reply:
done